topydo/lib/Colors.py

```diff
@@ -189,13 +193,19 @@ def ansicode(p_int, p_as_background=False):
         'underline': '4'
     }
 
-    decoration = decoration_dict[p_decoration]
-    fg_color = get_color(p_type, p_todo, p_256color)
-    bg_color = get_color(p_background, p_todo, p_256color) if p_background else -1
+    fg_color = get_color(p_type, p_todo)
+    bg_color = get_color(p_background, p_todo) if p_background else -1
+
+    try:
+        if 256 > fg_color >= 0:
+            decoration = decoration_dict[p_decoration]
+        else:
+            decoration = '0'
+    except TypeError:
```

Comment:
Did you mean `KeyError` here? Why not a `normal` decoration when a bogus decoration is passed to this function?